File: src/dotnet/ReSharperPlugin.TestingAssistant/Actions/GotoTestAction.cs
using JetBrains.Application.DataContext;
using JetBrains.Application.UI.ActionsRevised.Menu;
using JetBrains.Application.UI.ActionSystem.ActionsRevised.Menu;
using JetBrains.ReSharper.Feature.Services.Actions;
using JetBrains.ReSharper.Feature.Services.Navigation.ContextNavigation;
using ReSharperPlugin.TestingAssistant.Navigation;

namespace ReSharperPlugin.TestingAssistant.Actions
{
    [Action(Id, Name, Id = 17099)]
    public class GotoTestAction : ContextNavigationActionBase<LinkedTypesNavigationProvider>
    {
        public const string Id = "TestingAssistant.GotoTest";

        public const string Name = "Go to test";

        public override IActionRequirement GetRequirement(IDataContext dataContext)
        {
            return CurrentPsiFileRequirementNoCaches.FromDataContext(dataContext);
        }
    }
}

File: src/rider/main/kotlin/com/jetbrains/plugins/testingassistant/GotoTestAction.kt
package com.jetbrains.rider.plugins.testingassistant

import com.jetbrains.rider.actions.base.RiderAnAction
import icons.ReSharperIcons.UnitTesting

class GotoTestAction : RiderAnAction(
        "TestingAssistant.GotoTest",
        "Go to test",
        null,
        UnitTesting.TestFixtureToolWindow)